easybuild/easyblocks/c/cuda.py

```diff
@@ -266,6 +283,12 @@ def sanity_check_step(self):
             custom_paths['files'].append(os.path.join("extras", "CUPTI", "lib64", "libcupti.%s") % shlib_ext)
             custom_paths['dirs'].append(os.path.join("extras", "CUPTI", "include"))
 
+        # Just a subset of files are checked, since the whole list is likely to change,
+        # and irrelevant in most cases anyway
+        if os.path.exists(os.path.join(self.installdir, 'pkgconfig')):
+            pc_files = ['cublas.pc', 'cudart.pc', 'cuda.pc', 'nvidia-ml.pc', 'nvjpeg.pc']
```

Comment:
@damianam `nvidia-ml.pc` and `nvjpeg.pc` should be stripped out, or only added conditionally for recent versions, to avoid breaking the installation of old CUDA versions (like 7.x)

Reply:
fixed in c8f31cb